pkg/recorder/s3_prod.go

```diff
@@ -20,6 +20,7 @@ func (r *Recorder) upload() error {
 			r.conf.FileOutput.S3.Secret,
 			"",
 		),
+		Endpoint: aws.String(r.conf.FileOutput.S3.Endpoint),
```

Comment:
there should probably be a check for `r.conf.FileOutput.S3.Endpoint != ""` before setting this

Reply:
from https://github.com/aws/aws-sdk-go/blob/258263306d5faf0fc5f629dfcccf0f5ffedca131/aws/config.go#L44
	// An optional endpoint URL (hostname only or fully qualified URI)
	// that overrides the default generated endpoint for a client. Set this
	// to `nil` or the value to `""` to use the default generated endpoint.